Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -46,16 +53,54 @@ protected override void AnimateEye()
             // Base class animates the camera for crouching here, cinemachine handles that
         }
 
+        public override float GetBrakingDeceleration()
+        {
+            if (_slideBehaviour != null && _slideBehaviour.IsSliding)
+            {
+                return _slideBehaviour.brakingDecelerationSliding;
+            }
+            
+            return base.GetBrakingDeceleration();
+        }
+
+        public override float GetMaxSpeed()
+        {
+            if (_slideBehaviour != null && _slideBehaviour.IsSliding)
+            {
+                return _slideBehaviour.MaxWalkSpeedSliding;
+            }
+
+            return base.GetMaxSpeed();
+        }
```

Comment:
public should be above if Im not mistaken

Reply:
Moved it.